client/app/components/navigation/Bread.tsx

```diff
@@ -92,7 +92,7 @@ export default function Bread({
   // and useState which is maintained between renders of a top-level React component (required for next\back) navigations
   const searchParams = useSearchParams();
   const paramTitle = searchParams.get("title") as string;
-  const [crumbTitle, setCrumbTitle] = useState<string>("");
+  const [crumbTitle, setCrumbTitle] = useState<string>(paramTitle ?? "");
```

Comment:
This was unrelated to this PR, just adding this to the default state to hopefully stop the Happo breadcrumb loading diff since the useEffect below takes a second to run 🤞